### ISSUE TITLE
Different properties OR different relations in the same transaction

### ISSUE THREAD

**Issue by berrybeat** [berrybeat/Neo4j.Berries.OGM#66]
If two objects are updated, 1 property is set for one object and not for the other, e.g. company name OR the relation company - person, company - person KeepHistory=true, the person relation is created for one object and the person relation is not created for the other object, both relations are archived and only one is restored.

**Comment by farhadnowzari** [comment]
If it is regarding a `one to many` relation the archiving must not be handled here. It should be handled explicitly.

Let's say you have a company and a list of people. Now the KeepHistory is set to true on this list. What will happen is, if you update the company's people and only send one of them, the KeepHistory will archive all the others and only keep the one you sent. In this case you have to send the other ones two eventhough they were not changed.

Basically the `KeepHistory` should only be used on `1:*` relations if the use case is clear, otherwise the KeepHistory must not be used on such relations
